doc/features/build-arm-images.md

```diff
@@ -0,0 +1,210 @@
+    ```
+    GOARCH=arm64 CC=/opt/musl-cross/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc
+    ```
+
+    **Cross compile the aci-containers binaries:**
+
+    After updating the makefile for aci-containers repo, we can cross-compile as follows
+    ```
+    make all-static
+    ```
+
+2. **Update Dockerfiles:**
+
+    If a Dockerfile has a references to x86_64 libraries, we will replace them with similar repos for arm64 in both aci-containers repo and the opflex repo.
+    In the aci-containers and opflex repos please update the Dockerfiles under `docker/travis/` directory.
+    > Typically replacing x86_64 with aarch64 should work but please check if a similar package exists.
+
+    | x86_64                                               | arm64/aarch64                                               |
+    |------------------------------------------------------ | :-------------------------------------------------------------: |
+    | https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os | https://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os |
+    | https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os | https://mirror.stream.centos.org/9-stream/AppStream/aarch64/os |
+    | https://mirror.stream.centos.org/9-stream/CRB/x86_64/os | https://mirror.stream.centos.org/9-stream/CRB/aarch64/os |
+    | https://mirror.stream.centos.org/9-stream/AppStream/x86_64/debug/tree | https://mirror.stream.centos.org/9-stream/AppStream/aarch64/debug/tree |
+    | codeready-builder-for-rhel-8-x86_64-rpms             | codeready-builder-for-rhel-8-aarch64-rpms                   |
+    | kernel-headers-4.18.0-147.8.1.el8_1.x86_64           | kernel-headers-4.18.0-147.8.1.el8_1.aarch64.rpm             |
+    | kernel-devel-4.18.0-147.8.1.el8_1.x86_64             | kernel-devel-4.18.0-147.8.1.el8_1.aarch64.rpm               |
+    | kernel-modules-4.18.0-147.8.1.el8_1.x86_64           | kernel-modules-4.18.0-147.8.1.el8_1.aarch64.rpm             |
+    
+    **Cross build with docker buildx:**
+
+    Once the Dockerfiles have been updated they can be built as follows:
+    ```
+    docker buildx build --no-cache --platform linux/arm64 -t ${IMAGE_BUILD_REGISTRY}/<image>:${IMAGE_BUILD_TAG} --file=docker/travis/Dockerfile-<target> .
+    ```
+
+While the above steps explain the process to empower a user to build arm64 images themselves, the below step-by-step instructions can be followed to ensure a build on an `Ubuntu 22.04` machine.
+
+##  Instructions on building containers on `Ubuntu 22.04`
+
+###  Define Container Image Registry and Tag:
+    
+We will first define the image build registry and image tag, this will build all container images as
+`$IMAGE_BUILD_REGISTRY/<Image_Name>:IMAGE_BUILD_TAG`
+    
+    ```
+    export IMAGE_BUILD_REGISTRY="my_registry"
+    export IMAGE_BUILD_TAG="v0.1"
+    ```
```

Comment:
Add:

Replace "my_registry" and "v0.1" with the relevant registry name and tag in your environment.